x/bank/keeper/grpc_query.go

```diff
@@ -75,6 +75,48 @@ func (k BaseKeeper) AllBalances(ctx context.Context, req *types.QueryAllBalances
 	return &types.QueryAllBalancesResponse{Balances: balances, Pagination: pageRes}, nil
 }
 
+// SpendableBalances implements a gRPC query handler for retrieving an account's
+// spendable balances.
+func (k BaseKeeper) SpendableBalances(ctx context.Context, req *types.QuerySpendableBalancesRequest) (*types.QuerySpendableBalancesResponse, error) {
```

Comment:
This method name clobbers with the internal (public) API. I'd recommend moving the gRPC handler methods to a wrapper type, but that would be a breaking API change. I can do that in a separate PR s.t. we can still backport this one.